src/main/java/seedu/address/model/patient/Remark.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.patient;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Person's remark in the address book.
+ * Guarantees: immutable; is valid.
+ */
+public class Remark {
+
+    public static final String MESSAGE_CONSTRAINTS = "Remark can take any values, and it should not be blank";
```

Comment:
Yup I left it there as a placeholder as I'm still wondering if there are any forms an Invalid Remark can take. Any thoughts on this Jin Hao?

Reply:
For now, the only possible constraint I can think of is the word length, otherwise, a user can add a whole paragraph to a remark. What do you think?

Reply:
Yea good point! I'm able to limit the number of characters a user can add for a remark. However, I think the challenge with doing this may be the need to display the number of characters remaining(while the user is typing?) as it'll be hard for the user to manually count the number of characters added.